src/main/kotlin/org/rust/cargo/runconfig/ui/CargoCommandConfigurationEditor.kt

```diff
@@ -133,6 +132,12 @@ class CargoCommandConfigurationEditor(project: Project)
                 cargoProject()
             }
         }
+        row {
+            cell {
+                programInputRedirect.anchor()
+                programInputRedirect.component(pushX)
+            }
```

Comment:
Not sure how to fix it.

Reply:
looks like `cell(isFullWidth = true)` should help

Reply:
It doesn't work well in fullscreen:

![image](https://user-images.githubusercontent.com/6079006/100517288-a5b8ac00-319a-11eb-89e0-7b26729d16df.png)

Reply:
Looks like you should replace `.apply { makeWide() }()` with `(CCFlags.growX)` for env variables and working directory components to fix it